src/models/health.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const db = require('../lib/db')
+
+function * getDbHealth () {
+  const isConnected = yield db.isConnected()
+  return isConnected ? {
+    status: 'OK'
+  } : {
+    status: 'KO'
```

Comment:
Is this a convention that we've decided on? I'd prefer something more clear and standard.